cluster/config-defaults.yaml

```diff
@@ -279,3 +279,6 @@ vm_dirty_bytes: ""
 
 # temporary flag for kubernetes.io/node-pool node label
 legacy_node_pool_label_enabled: "false"
+
+# default ttl for kube janitor for resources build from PRs in namespaces matching .*-pr-.*
+default_pr_ttl: "1w"  # 1 week
```

Comment:
could you prefix this config key with: `kube_janitor` so it's easier to understand what it's used for.